### ISSUE TITLE
Incorrect data collection for wireless interfaces with same name.

### ISSUE THREAD

**Issue by dburianov** [akpw/mktxp#123]
Follow by recomendation I have same name for wireless interfaces in 2.4 and 5. In this situation mktxp export value with same labels but with different value. e.g.
`
mktxp_interface_rx_byte_total{name="High",routerboard_address="10.0.0.254",routerboard_name="w"} 8.729986406e+09
mktxp_interface_rx_byte_total{name="High",routerboard_address="10.0.0.254",routerboard_name="w"} 2.0484126092e+010
mktxp_interface_tx_byte_total{name="UA",routerboard_address="10.0.0.254",routerboard_name="w"} 8.065296e+06
mktxp_interface_tx_byte_total{name="UA",routerboard_address="10.0.0.254",routerboard_name="w"} 7.2106699158e+010
` 
In this situation we can not make any post processing for this metrics.
As solve for this issue add new label for this metrics line interface="wlan10"

**Comment by akpw** [comment]
just curious about that "recommendation", the higher numbers are from the 2.4 wifi? Given it's broader coverage range, wouldn't the chances be for your devices just hanging there rather than ever utilising the 5 GHz band?

**Comment by dburianov** [comment]
follow by statictic on routers 75% devices use 5GHz. This devices switched to 2.4GHz on case when signal level of 5 worst than 2.4. Same situation when we use single router or CapMan or any other brands.

**Comment by akpw** [comment]
still a bit confused here, are you talking about the same SSIDs or the same interface name (which is what gets actually exported)?

**Comment by dburianov** [comment]
I say that I have same SSIDs on 2 wlan interfaces 2.4 and 5.
Now mktxp export next metrics:
`
mktxp_interface_rx_byte_total{name="High",routerboard_address="10.0.0.254",routerboard_name="w"} 8.729986406e+09 mktxp_interface_rx_byte_total{name="High",routerboard_address="10.0.0.254",routerboard_name="w"} 2.0484126092e+010 mktxp_interface_tx_byte_total{name="UA",routerboard_address="10.0.0.254",routerboard_name="w"} 8.065296e+06 mktxp_interface_tx_byte_total{name="UA",routerboard_address="10.0.0.254",routerboard_name="w"} 7.2106699158e+010
`

with this metrics I see that SSID High or UA have traffic 9-60Gbs
![mktxp](https://github.com/akpw/mktxp/assets/3417811/3b4c6d0f-ed8a-4d85-ac9a-161b22d5b8cb)

as options for fix it please add to the metrics label like  **interface="wlanX"** e.g.
`
mktxp_interface_rx_byte_total{name="High",routerboard_address="10.0.0.254",routerboard_name="w", interface="wlan0"} 8.729986406e+09
mktxp_interface_rx_byte_total{name="High",routerboard_address="10.0.0.254",routerboard_name="w", interface="wlan1"} 2.0484126092e+010
mktxp_interface_tx_byte_total{name="UA",routerboard_address="10.0.0.254",routerboard_name="w", interface="wlan2"} 8.065296e+06
mktxp_interface_tx_byte_total{name="UA",routerboard_address="10.0.0.254",routerboard_name="w", interface="wlan03"} 7.2106699158e+010
`

Thank you

**Comment by akpw** [comment]
well... as mentioned in the previous comment, these are interface rx / tx metrics and should have nothing to do with the SSIDs naming. Looking at the picture, the in/out metric for your interfaces seems to make sense as well.

**Comment by dburianov** [comment]
I agree, it feature from RouterOS. we can not identify interface by id.
